cms/src/main/java/uk/nhs/hee/web/validation/validator/MandatoryPublicationTypeValidator.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.hee.web.validation.validator;
+
+import uk.nhs.hee.web.constants.HEETaxonomyNodeNames;
+
+import java.util.Arrays;
+
+/**
+ * Validates that a publication type has been supplied (it's not enforced through the interface
+ * as it's a taxonomy field)
+ */
+public class MandatoryPublicationTypeValidator extends AbstractMandatoryValidator {
```

Comment:
Could we replace this validator with OOTB `required` validator (e.g. `/hippo:namespaces/hee/publicationLandingPage/hipposysedit:nodetype/hipposysedit:nodetype/globalTaxonomyPublicationTypes/hipposysedit:validators: [required]`) if possible ?

Reply:
I will add this. It didn't work in 15.2.x so I had kept teh old method, but I'll add it now !